client/src/components/JsonViewer.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <Json v-for="(item, key) in json" :name="key" :value="item" :open="true" />
+</template>
+
+<script>
+import Json from "./Json.vue";
+import emitter from "tiny-emitter/instance";
+import axios from "axios";
```

Comment:
Total ne blöde Frage aber warum eigentlich Axios? Ich glaube ich benutze es sogar auch in meinem Projekt aber ist die fetch-Api nicht inzwischen (gerade für diesen Fall) absolut ausreichend?

Reply:
Weiß nicht wie ich die Frage als "nicht nötig für diesen PR" markieren kann. Sorry. :)

Reply:
Ja fetch api wäre vollkommen ausreichend. Axios nur aus Bequemlichkeit und aus der Angst heraus das die Anforderungen sich so verändern das fetch unnötig kompliziert wird.